net/gce.sh

```diff
@@ -705,6 +718,22 @@ $(printNetworkInfo)
 $(creationInfo)
 EOM
 
+metadata_req() {
```

Comment:
Can you move this into a `net/script/` instead of inlining, this'll reduce all the escaping.

We can write `$selfDestructMinutes` to a file in `/` on the instance first, which'll could allow us to also augment the solana user's prompt to include the amount of time remaining so users know how much time they have left.

Also if self-destruct is enabled, let's add that fact to `motd` to again notify the user.  

Finally it'd be nice to support aborting the self destruct, perhaps by removing the file in `/` containing `$selfDestructMinutes`

Reply:
Yes!  I was over here holding myself off of adding the bells and whistles for the sake of getting it in 😅

Reply:
Yeah lets show our fellow devs a little ❤️ on this one.  No bad karma